lua/midi.lua

```diff
@@ -78,6 +146,14 @@ norns.midi.event = function(id, data)
     if d.event ~= nil then
       d.event(data)
     end
+    -- do any individual subscribed callbacks
+    for name,callback in pairs(Midi.devices[id].callbacks) do
```

Comment:
again maybe: `for _, callback in pairs ...` ? (also note that `name` here shadows `name` in the function scope.)